indico/modules/rb_new/client/js/common/roomSearch/reducers.js

```diff
@@ -69,6 +69,7 @@ export function initialRoomFilterStateFactory(namespace) {
         Object.assign(state, {
             myBookings: false,
             hideUnused: false,
+            showCancelled: false,
```

Comment:
`showInactive` to cover both cancelled/rejected? the current name indicate it's only the former...

Reply:
Thought about sth the same but actually wanted to wait for some opinions first before changing it